Documentation/package_Windows.md

```diff
@@ -0,0 +1,61 @@
+## Runtime requirements
+
+A Windows game targeting standard desktop Windows will require the following in order to run:
+# Windows 7 or above (Windows XP and Vista are unsupported)
+# DirectX 11 compliant graphics card and driver
+# DirectX June 2010 redistributables
+# .Net Framework 4.5
+# Windows Media Player
+
+## Handling the Windows Media Player dependency
+
+While most store automatically manage the .Net and DirectX dependencies, they don't verify the presence of Windows Media Player.
+
+Since it is forbidden by its license to redistribute it in any form, it is best to implement a check in your game and to prompt the users to install Windows Media Player if it isn't present. To do so, you can use the following code in your Program.cs file (before Game.Run()):
+
+#if WINDOWS
```

Comment:
@tomspilman can we do this any other way? or add it to the framework (without the WinForms dependency) seems a pain. Or have a templates nsis installer which does the checks?